src/webgpu/api/operation/render_pass/resolve.spec.ts

```diff
@@ -0,0 +1,206 @@
+export const description = `API Operation Tests for RenderPass StoreOp.
+
+Tests a render pass with a resolveTarget resolves correctly for many combinations of:
+  - number of color attachments, some with and some without a resolveTarget
+  - renderPass storeOp set to {‘store’, ‘clear’}
+  - resolveTarget mip level set to {‘0’, base mip > ‘0’}
+  - resolveTarget base array layer set to {‘0’, base layer > '0'} for 2D textures
+  TODO: test all renderable color formats
+`;
+
+import { params, poptions } from '../../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+
+// Test with a zero and non-zero base mip level.
+const kBaseMipLevel: number[] = [0, 1];
```

Comment:
I think it would be better to inline these lists of options in the `poptions` calls, to put them closer to where they're used. Unless you intend to use these more than once and really want them to stay in sync.

Reply:
Done.